packages/slice-machine/components/DeleteSliceModal/index.tsx

```diff
@@ -17,7 +18,7 @@ type DeleteSliceModalProps = {
 
 export const DeleteSliceModal: React.FunctionComponent<
   DeleteSliceModalProps
-> = ({ sliceName, libName }) => {
+> = ({ sliceId, sliceName, libName }) => {
```

Comment:
Do we need `sliceName`? Can't see it used

Reply:
Yes, it's for the description in the modal